cypress/support/generateUser.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+const { faker } = require('@faker-js/faker')
+
+function generateUser() {
+  const userName = faker.internet.userName()
+  const email = faker.internet.email()
+  const mobile = Math.random().toString().slice(2, 12)
```

Comment:
The mobile number is generated using `Math.random()`, which may lead to invalid or inconsistent numbers. Consider using `faker.phone.phoneNumber()` to generate more realistic and valid phone numbers.